packages/arb-token-bridge-ui/src/util/wagmi/wagmiAdditionalNetworks.ts

```diff
@@ -142,6 +142,27 @@ export const localL2Network: Chain = {
     }
   },
   blockExplorers: {
-    default: { name: 'Blockscout', url: '' }
+    default: { name: 'Blockscout', url: 'https://etherscan.io' }
```

Comment:
Metamask now requires block explorer url, otherwise it throws in the bridge UI if you try to add it locally first time